UnitTests/UnitTests.HighPerformance.Shared/Extensions/Test_ArrayExtensions.2D.cs

```diff
@@ -120,30 +121,11 @@ public void Test_ArrayExtensions_2D_FillArrayTwice()
 
         [TestCategory("ArrayExtensions")]
         [TestMethod]
-        public void Test_ArrayExtensions_2D_FillArrayNegativeSize()
+        public void Test_ArrayExtensions_2D_AsSpan2DAndFillArrayBottomEdgeBoundary()
         {
             bool[,] test = new bool[4, 5];
 
-            test.Fill(true, 3, 4, -3, -2);
-
-            var expected = new[,]
-            {
-                { false, false, false, false, false },
-                { false, false, false, false, false },
-                { false, false, false, false, false },
-                { false, false, false, false, false },
-            };
-
-            CollectionAssert.AreEqual(expected, test);
-        }
-
-        [TestCategory("ArrayExtensions")]
-        [TestMethod]
-        public void Test_ArrayExtensions_2D_FillArrayBottomEdgeBoundary()
```

Comment:
Did some of these tests get removed or are they integrated in other scenarios now?

Reply:
Those tests are gone as that API has also been removed to streamline the API surface and make it more consistent with the BCL as well - the fill functionality is now available through the `Span2D<T>` type (just like you can fill an array through the `Span<T>` type today), and the tests are here (just a few, there's more in other places as well):

https://github.com/windows-toolkit/WindowsCommunityToolkit/blob/b75f2381adbc45207b8258dc21cf432eaadc7e2d/UnitTests/UnitTests.HighPerformance.Shared/Extensions/Test_ArrayExtensions.2D.cs#L87-L89

As for usage, the refactoring needed is pretty simple:

```csharp
// Before
array.Fill(1, 1, 3, 2, true);

// After
array.AsSpan2D(1, 1, 2, 3).Fill(true);
```

This also helps to separate the logic between slicing the target area and actually filling it, and it also means we can reutilize the same code to work on a number of different scenarios, since this will now work with just a target `Span<T>` instance, which itself could map anything from a 2D array, to a 1D array, to some native memory as well 😄